File: src/test/java/koorkah/TestUpper.java
package koorkah;

import org.junit.jupiter.api.Test;

class TestUpper extends BaseT
{

    @Test
    void test()
    {
        String expression = koorkah.toRegEx("upper");
        assertRegexIs("\\p{Upper}", expression);
        assertRegexMatchesText(expression, "A");
        assertRegexDoesntMatchText(expression, "a");
    }

}

File: src/python/test/TestUpper.py
import unittest
from Utils import Utils 
from Refi import Refi


class TestUpper(unittest.TestCase):

    def setUp(self):
        self.refi = Refi()
        self.utils = Utils()


    def test_(self):

        expression = self.refi.toRegEx("upper")
        self.utils.assertRegexIs("\\p{Upper}", expression)
        self.utils.assertRegexMatchesText(expression, "A")
        self.utils.assertRegexDoesntMatchText(expression, "a")

    def runTest(self):
        pass  

if __name__ == '__main__':
    unittest.main()    
    


